MapboxNavigation/RouteMapViewController.swift

```diff
@@ -682,31 +682,12 @@ extension RouteMapViewController: NavigationViewDelegate {
     }
 
     func labelCurrentRoadFeature(at location: CLLocation) {
-        guard let style = mapView.style, let stepCoordinates = routeController.routeProgress.currentLegProgress.currentStep.coordinates else {
+        guard let stepCoordinates = routeController.routeProgress.currentLegProgress.currentStep.coordinates else {
             return
         }
 
         let closestCoordinate = location.coordinate
         let roadLabelLayerIdentifier = "roadLabelLayer"
-        var streetsSources: [MLNVectorTileSource] = style.sources.compactMap {
-            $0 as? MLNVectorTileSource
-        }.filter(\.isMapboxStreets)
-
-        // Add Mapbox Streets if the map does not already have it
-        if streetsSources.isEmpty {
```

Comment:
Do you expect that there are valid reasons for the person to be in this state where they would want to silently proceed without error?

If not, though we don't have a reasonable default layer to fall back to like mapbox did, could we instead help the user out with an assertionFailure or a log message here?